Funcky.SourceGenerator/Funcky.SourceGenerator.csproj

```diff
@@ -7,11 +7,11 @@
         <Nullable>enable</Nullable>
         <LangVersion>preview</LangVersion>
         <IsPackable>false</IsPackable>
+        <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
     </PropertyGroup>
-
     <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" PrivateAssets="all" />
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" />
+        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" PrivateAssets="all" VersionOverride="3.11.0" />
```

Comment:
I chose to go with a version override here as to not break compatibility with older visual studio versions for our public analyzers.